lib/__tests__/_algoliaAgent.test.ts

```diff
@@ -1,4 +1,4 @@
-import AlgoliaInsights from "../insights";
+import AlgoliaInsights from "../entryBrowser";
```

Comment:
I think we should keep the tests on `insights`, even if it means that we have to instantiate the class ourselves. At least we ensure that we don't test any environment specific with the unit test. Both entry points can be covered by integration tests that runs inside each specific environment.

Reply:
I agree with that. I was having the same concern this morning :) 
https://github.com/algolia/search-insights.js/pull/140/commits/95ec9aaa08c9b2726a1e119c5a164545e5250671